src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,9 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+import mate.academy.model.User;
+
+public class AuthenticationService extends UserService {
```

Comment:
```suggestion
public class AuthenticationService {
```
Don’t use inheritance here. Create class-level variable 
` private final UserService userService = new UserService();`